orchestrator/redisClientWrapper.js

```diff
@@ -31,38 +30,37 @@ class ClientWrapper {
             return;
           }
 
-          this.requestTemplateList(tenant, deviceid, resolve, reject, redisState);
+          this.requestDeviceInfo(tenant, deviceid, resolve, reject, redisState);
         }).catch(() => {
           console.log(`[redis] Could not get from redis, will request to device manager`);
-          this.requestTemplateList(tenant, deviceid, resolve, reject, redisState);
+          this.requestDeviceInfo(tenant, deviceid, resolve, reject, redisState);
           return;
         });
       } else {
-        this.requestTemplateList(tenant, deviceid, resolve, reject, redisState);
+        this.requestDeviceInfo(tenant, deviceid, resolve, reject, redisState);
       }
     });
   }
 
   /**
-   * Requests template list to devicemanager.
+   * Requests device info to devicemanager.
    * @param {string} deviceid 
    * @param {string} tenant 
    * @param {callback} resolve 
    * @param {callback} reject 
    */
-  requestTemplateList(tenant, deviceid, resolve, reject, redisState) {
+  requestDeviceInfo(tenant, deviceid, resolve, reject, redisState) {
```

Comment:
There is some reason to don't return a Promise instead use these parameters?

Reply:
Forget it, this behavior is present in other parts, I think it is out of this PR's scope deal with it.

Excellent work.

Reply:
That's ok! Thanks =)